alignak/dispatcher.py

```diff
@@ -473,7 +478,12 @@ def prepare_dispatch_schedulers(self):
                     conf.is_assigned = True
                     conf.assigned_to = sched
 
-                    # We update all data for this scheduler
+                    # We updated all data for this scheduler
+                    pickled_conf = cPickle.dumps(conf_package)
+                    logger.info('[%s] scheduler configuration %s size: %d bytes',
+                                realm.get_name(), sched.scheduler_name, sys.getsizeof(pickled_conf))
```

Comment:
Because it is not a correct and accurate size! Pickling the data makes sure that you have the siez of all the data ... as I noticed in my tests and found on StackOverflow ...